tensorflow_probability/python/experimental/mcmc/covariance_reducer.py

```diff
@@ -229,7 +258,7 @@ def finalize(self, final_reducer_state):
 
     Returns:
       covariance: an estimate of the covariance with identical structure to
```

Comment:
Identical structure to the _results_ of `self.transform_fn`, here and elsewhere.

To illustrate the difference, if `self.transform_fn = lambda x: [x, x+1]`, this code should compute the covariance of both `x` and `x+1`, even though there was nominally only one `transform_fn`.